packages/framework-provider-azure-infrastructure/src/infrastructure/rockets/infrastructure-rocket.ts

```diff
@@ -0,0 +1,43 @@
+import { BoosterConfig, RocketDescriptor } from '@boostercloud/framework-types'
+import { FunctionDefinition } from '../types/functionDefinition'
+import { ApplicationSynthStack } from '../types/application-synth-stack'
+import { RocketUtils } from './rocket-utils'
+
+export interface InfrastructureRocket {
+  packageName?: string
+  mountStack: (
+    config: BoosterConfig,
+    applicationSynthStack: ApplicationSynthStack,
+    utils: RocketUtils
+  ) => ApplicationSynthStack
+  unmountStack?: () => void
+  getFunctionAppName?: (applicationSynthStack: ApplicationSynthStack) => string
+  mountFunctions?: (
+    config: BoosterConfig,
+    applicationSynthStack: ApplicationSynthStack,
+    utils: RocketUtils
+  ) => Array<FunctionDefinition>
+}
+
+function requireRocket(rocketPackageName: string): (params: unknown) => InfrastructureRocket {
+  return require(rocketPackageName).default
+}
+
+export function loadRocket(rocketDescriptor: RocketDescriptor): InfrastructureRocket {
+  const rocketBuilder = requireRocket(rocketDescriptor.packageName)
+  if (!rocketBuilder)
+    throw new Error(
+      `Could not load the rocket package ${rocketDescriptor.packageName}. Make sure you installed it in your project devDependencies.`
```

Comment:
Now it is more accurate to say "... the rocket _infrastructure_ package...", as the rocket core package must be in the normal "dependencies" section.

Maybe it is this way in the AWS provider, but it should be changed too
```suggestion
      `Could not load the rocket infrastructure package ${rocketDescriptor.packageName}. Make sure you installed it in your project devDependencies.`
```